advisor/src/test/assets/ort-analyzer-result.yml

```diff
@@ -42,8 +42,8 @@ analyzer:
     ort_version: "HEAD"
     java_version: "11.0.9"
     os: "Linux"
-    processors: "4"
-    max_memory: "8192"
+    processors: 4
```

Comment:
Commit message: Maybe better explicitly say "ORT *result* files", as this doesn't reserialize e.g. serialized `ProjectAnalyerResult`s, IIUC.

Typo: s/algin/align/